t/20-openqa-isotovideo-utils.t

```diff
@@ -67,6 +68,19 @@ subtest 'error handling when loading test schedule' => sub {
     };
 };
 
+subtest 'prevent upload assets when publish_hdd is none with case-insensitive' => sub {
+    my $command_handler = OpenQA::Isotovideo::CommandHandler->new();
+    my @possible_values = qw(none None NONE);
+    $bmwqemu::vars{"BACKEND"} = 'qemu';
+    for my $v (@possible_values) {
+        $bmwqemu::vars{PUBLISH_HDD_1} = $v;
+        $command_handler->test_completed(1);
+        my $return_code;
+        my $log = combined_from { $return_code = handle_generated_assets($command_handler, 1) };
+        like $log, qr/Asset upload is skipped for PUBLISH_HDD/, "Upload is skipped when PUBLISH_HDD_1 is $v";
+    }
+};
```

Comment:
I think the whole "publish assets" subtest from 14-isotovideo.t  should be moved here together.
It doesn't make sense to have very similar tests in two different test scripts.

Reply:
To my mind both are not great. I'd say leave it where it is for now - or actually move this and other asset tests into a new file

Reply:
i agree as i notice but i didnt want to "take it too far". please instruct me how to proceed. or we can merge this and create another one as anyway it is going to be another commit anyway

Reply:
nevermind. Just fix the author test failure :)

Reply:
@b10n1k This:

```
The following tests FAILED:
	  5 - test-local-perl-style (Failed)
```

Reply:
yea but why? the tidy doesnt do anything

Reply:
solved